projects/igniteui-angular/src/lib/grids/grid-base.component.ts

```diff
@@ -1688,6 +1699,7 @@ export abstract class IgxGridBaseComponent extends DisplayDensityBase implements
         return this.showToolbar &&
```

Comment:
Can we simplify this Boolean expression?

Reply:
@rmkrmk I agree with @rkaraivanov. Just show the toolbar every time `this.showToolbar === true`, even if no elements would be displayed in it.